src/loaddb/load_db_value_converter.cpp

```diff
@@ -161,6 +161,11 @@ namespace cubload
     setters_[DB_TYPE_CLOB][LDR_ELO_INT] = &to_db_elo_int;
 
     setters_[DB_TYPE_JSON][LDR_STR] = &to_db_json;
+
+    setters_[DB_TYPE_MONETARY][LDR_INT] = &to_db_monetary;
```

Comment:
SA loaddb does mapping only between DB_TYPE_MONETARY and LDR_MONETARY
I see two solutions here either we support only this mapping either we need to new mappings also on SA

Reply:
SA loader works differently. 
It has generic handlers:
```c
  attdesc->setter[LDR_NULL] = &ldr_null_db_generic;
  attdesc->setter[LDR_INT] = &ldr_int_db_generic;
  attdesc->setter[LDR_NUMERIC] = &ldr_numeric_db_generic;
  attdesc->setter[LDR_DOUBLE] = &ldr_real_db_generic;
  attdesc->setter[LDR_FLOAT] = &ldr_real_db_generic;
```
They enable int, numeric and floating points to be loaded into monetary.

Reply:
Ok, somehow I missed that